delete_me.py

```diff
@@ -0,0 +1,3 @@
+good_var_name = 1
+very_bod_vor_name = 1
+not_so_bad_nome = 1
```

Comment:
format: typos:
Possible typos in `not_so_bad_nome`, suggestions:
Candidate #1 with a confidence 36%
```suggestion
not_so_bad_some = 1
```
Candidate #2 with a confidence 32%
```suggestion
not_so_bad_none = 1
```
Candidate #3 with a confidence 31%
```suggestion
not_so_bad_name = 1
```